### PR TITLE
Visually displayed copyright lang attribute

### LINKED ISSUE #623: Visually displayed copyright text requires lang attribute
**Is your feature request related to a problem? Please describe.**
The visually displayed copyright text is always in English 

**Describe the solution you'd like**
* On non-English articles we should add a `lang` attribute to the visually displayed copyright text so this can be announced by screen readers with the correct accent. This should not be applied to the visually hidden text 'Image source, '.
* In Persian we expect the copyright span to be right aligned.

Arises from https://github.com/bbc/simorgh/pull/618

**Testing notes**
English News article example: http://localhost:7080/news/articles/cn7769kpk9mo
Persian article example: http://localhost:7080/persian/articles/cyddjz5058wo

* Screen reader - test both the visually hidden text such as: 'Image source, ' and the visually displayed copyright text such as 'Getty images' are read with the correct accents - Make sure to check that this works ok with `role="text"`
* Check to make sure that the visually hidden text 'Image source, ' (translated) is read with a Farsi accent on `Persian`
* Check to make sure that the visually displayed copyright text such as 'Getty images' is read with an English accent on `Persian`

Dev insight: We should have unit tests for the changes.

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves [#623](https://github.com/bbc/simorgh/issues/623)

**Overall change:** _Adds a default lang attribute ie "en-gb" to ensure visually hidden text such as: 'Image source, ' and the visually displayed copyright text such as 'Getty images' are read with the correct accents"_

**Code changes:**

- _Adds a lang attribute to copyright text_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/containers/Copyright/__snapshots__/index.test.jsx.snap
Comment (sareh) [inline src/app/containers/Copyright/__snapshots__/index.test.jsx.snap]: The original text here for the Persian example is incorrect. The copyright text will always be in English, even for the Persian service. 
```suggestion
    Getty Images
```

File: src/app/containers/Copyright/index.jsx
Comment (sareh) [inline src/app/containers/Copyright/index.jsx]:
```suggestion
      <span lang="en-GB">{children}</span>
```
We should only add the lang attribute of `en-GB` here if the article's language value is not `en-GB`. 

We can add the language to the Service configuration files and use the value from there.
e.g. 
the News config can have `lang: 'en-GB',` added https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/news.js#L6
likewise the Persian config can have `lang: 'fa',` added
https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/persian.js#L6

This lang attribute for the article can be fetched via useContext avoce, just like the `imageCopyrightOffscreenText` is currently pulled in. 

Then we'll only add `lang="en-GB"` if this language value is not already `en-GB`.

**Review by sareh — CHANGES_REQUESTED** [review]
Looks good so far. I've added a comment about only adding the attribute when it is required.

File: src/app/containers/Copyright/index.jsx
Comment (RayNjeri) [reply src/app/containers/Copyright/index.jsx]: @sareh should I then create langContext to fit this case?

File: src/app/lib/config/services/news.js
Comment (sareh) [inline src/app/lib/config/services/news.js]:
```suggestion
  lang: 'en-GB',
```
The lang value should have a dash rather than an underscore, as in the previous comments. Everything else here looks good.

**Review by sareh — CHANGES_REQUESTED** [review]

**Review by sareh — DISMISSED** [review]
Looked at it locally with this Persian article: http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o & this News article: http://localhost.bbc.com:7080/news/articles/c0g992jmmkko
Looks good! 
When it has two approvals, this ticket will move over into the Accessibility review column of the board - here we can test this with a screenreader. Description of how to use VoiceOver on a Mac is here: https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os#shortcut-keys

@greenc05 has written the criteria for accessibility testing this change in Testing notes in the issue description.

File: src/app/containers/Copyright/index.jsx
Comment (dr3) [inline src/app/containers/Copyright/index.jsx]: As far as I can see, theres no snapshot that tests this functionality, can you add one please?

(a test that uses a service context where lang === `en-GB` to see the snapshot without the `lang` on the span)

**Review by dr3 — DISMISSED** [review]

**Review by 12 — COMMENTED** [review]
Code looks good to me and is functionally working locally. Nothing extra to add, just what @dr3 has already highlighted, will approve once that's done.

File: src/app/containers/Copyright/index.test.jsx
Comment (sareh) [inline src/app/containers/Copyright/index.test.jsx]: I believe this should have a lang of farsi
```suggestion
  lang: 'fa',
```

File: src/app/containers/Copyright/index.test.jsx
Comment (sareh) [inline src/app/containers/Copyright/index.test.jsx:15]: Likewise for the test it'd be good to add `lang: 'en-GB'` here in the News Service context stub.

**Review by sareh — APPROVED** [review]
👍 The test update looks good.

**Review by 12 — APPROVED** [review]

**Comment by sareh** [comment]
Update on accessibility testing with @RayNjeri. 
On Mac with Voiceover - we can download different synthesised voices locally e.g. for English, French, Spanish, Russian, etc, but the screenreader is not able to switch between different voices automatically upon reaching the appropriate `lang` attribute that matches it.

On a Windows machine with JAWS - with supported `lang` attribute, the screenreader automatically switches to use the voice of the language, if it is loaded on the machine. If not, it defaults to stating `Russian`, for example, if `lang="ru"`. 

Discussing this with @greenc05 and reading up online about this, this is expected behaviour. So putting this ticket in 'Ready for test'.

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**
